broker/lib/fabrik/UnlockResourcePoller.js

```diff
@@ -18,7 +19,12 @@ class UnlockResourcePoller {
           const resourceState = resource.body.status.state;
           logger.debug(`[Unlock Poller] Got resource ${lockDetails.lockedResourceDetails.resourceId} state as `, resourceState);
           //TODO-PR - reuse util method is operationCompleted.
-          if (resourceState === CONST.APISERVER.RESOURCE_STATE.SUCCEEDED || resourceState === CONST.APISERVER.RESOURCE_STATE.FAILED || resourceState === CONST.APISERVER.RESOURCE_STATE.ERROR) {
+          if (_.includes([
+              CONST.APISERVER.RESOURCE_STATE.SUCCEEDED,
+              CONST.APISERVER.RESOURCE_STATE.FAILED,
+              CONST.APISERVER.RESOURCE_STATE.ERROR,
```

Comment:
I see that [here](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/337/files#diff-97bb265f5747f3062215bfee484e6d48L252) `ERROR` is removed from resource state , then is this required ?

Reply:
Removed